app/scripts/controllers/pod.js

```diff
@@ -139,8 +139,8 @@ angular.module('openshiftConsole')
         }
         var windowWidth = win.width();
         var windowHeight = win.height();
-        var termWidth = windowWidth - r.left - 40; // we want 40px right padding, includes 20px padding within the container terminal
-        var termHeight = windowHeight - r.top - 50; // we want 50px bottom padding, includes 20px padding within the container terminal
+        var termWidth = windowWidth - r.left - 54; // we want 40px right padding
+        var termHeight = windowHeight - r.top - 36; // we want 36px bottom padding, includes 6px border within the container terminal
```

Comment:
the existing borders actually ended up allowing for really small gutters at certain sizes, this gives better spacing and takes into account the new terminal styles